lib/class-wp-theme-json.php

```diff
@@ -251,7 +251,7 @@ class WP_Theme_JSON {
 	);
 
 	const ELEMENTS = array(
-		'link' => 'a:not(.wp-block-button_link)',
```

Comment:
It definitely makes sense to avoid tying the elements selector to specific block selectors.

If we do find an issue with the change, we may want to consider adding specificity to the block itself.